Xamarin.Forms.Core/FontAttributes.cs

```diff
@@ -3,10 +3,39 @@
 namespace Xamarin.Forms
 {
 	[Flags]
+	[TypeConverter(typeof(FontAttributesConverter))]
 	public enum FontAttributes
 	{
 		None = 0,
 		Bold = 1 << 0,
 		Italic = 1 << 1
 	}
+
+	[Xaml.TypeConversion(typeof(FontAttributes))]
+	public sealed class FontAttributesConverter : TypeConverter
+	{
+
+		public override object ConvertFromInvariantString(string value)
+		{
+			if (string.IsNullOrEmpty(value))
+				return FontAttributes.None;
+
+			FontAttributes attributes = FontAttributes.None;
+			for (int i = 0; i < value.Split(',').Length; i++) {
```

Comment:
Lets not perform the split 3 different times unless there is a reason to I dont see?

Reply:
auto-refactoring the `foreach` into a `for` did this